lib/measured/measurable.rb

```diff
@@ -59,6 +59,10 @@ def units
       conversion.unit_names
     end
 
+    def valid_unit?(unit)
+      conversion.unit_or_alias?(unit)
```

Comment:
No. This is dealt with in `unit_or_alias?`. This is basically just a delegation, but with a cleaner name presented in the public API.